examples/src/Examples.js

```diff
@@ -11,6 +11,8 @@ import MultipleSections from './MultipleSections/MultipleSections';
 import EventsPlayground from './EventsPlayground/EventsPlayground';
 import EventsLog from './EventsLog/EventsLog';
 
+let eventQueue = [];
```

Comment:
I'd prefer to have `eventQueue` in the class itself (the same way we have `eventsPlaceholder`).